mlflow/ml-package-versions.yml

```diff
@@ -360,8 +360,8 @@ spark:
       cd $temp_dir
       # -Pconnect -Phive is required by spark connect
       ./build/sbt -Pconnect -Phive clean package connect/assembly
-      cd python
-      pip install '.[connect]'
+      cd python/packaging/connect
+      pip install .
```

Comment:
does this only install `pyspark-connect`?

Reply:
i think it does, updated